src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+    const content: any
```

Comment:
If you wanna avoid using `any` you can declare the type like here (https://duncanleung.com/typescript-module-declearation-svg-img-assets/). But it's still is a string after being imported, so wouldn't be that useful for type checking. Not sure though if it's better than using `any` 🤷🏼

Reply:
In most of documentations even in official gatsby they just use `any` and like what you said @khalid-deriv  It's not that very useful even if we are able to add a proper type for it. I think we can keep it as `any`